lib/vmdb/loggers.rb

```diff
@@ -26,6 +26,7 @@ def self.apply_config(config)
       apply_config_value(config, $kube_log,          :level_kube)
       apply_config_value(config, $mw_log,            :level_mw)
       apply_config_value(config, $datawarehouse_log, :level_datawarehouse)
+      apply_config_value(config, $monitoring_log,    :level_monitoring)
```

Comment:
This seems a little too broad for a logging category (same comment for the logger definition below).

If we're introducing the concept of a Monitor, I could easily see monitoring as a concept spanning multiple providers (OpenStack Gnocci/Ceilometer or AWS CloudWatch for instance).  If we refactor some of the other providers to have "monitoring" instead of just "metrics", then I wouldn't want to have a single logger category for that.

In the same way I wouldn't want to have a single "inventory.log" file.

Maybe a better idea here would be "container_monitoring"?  I can see why getting more specific with "prometheus_monitoring" or "hawkular_monitoring" is less desirable, since the possibility of reusing those specific services in different providers.

/cc @Fryguy @jrafanie @gtanzillo 

Thoughts?